docarray/utils/_internal/misc.py

```diff
@@ -77,6 +84,10 @@ def is_tf_available():
     return tf_imported
 
 
+def is_jnp_available():
```

Comment:
```suggestion
def is_jax_available():
```

lets rename it to this